woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -721,7 +721,7 @@ def test_underlying_index(sample_df):
     assert schema_df.index.name is None
     assert type(schema_df.index) == specified_index
 
-    # --> add back when df.ww.drop is implemented
+    # --> add back when we figure out how to handle removing indices
```

Comment:
Is this need covered by an issue already?

Reply:
Yeah, that's a fair point. Can probably just delete this commented out code block since it'll get handled by that issue.